applications/StructuralMechanicsApplication/python_scripts/distribute_force_on_surface_process.py

```diff
@@ -0,0 +1,64 @@
+import KratosMultiphysics as KM
+import KratosMultiphysics.StructuralMechanicsApplication as KSM
+
+
+def Factory(settings, Model):
+    if not isinstance(settings, KM.Parameters):
+        raise Exception("expected input shall be a Parameters object, encapsulating a json string")
+    return DistributeForceOnSurfaceProcess(Model, settings["Parameters"])
+
+
+class DistributeForceOnSurfaceProcess(KM.Process):
```

Comment:
```suggestion
class DistributeLoadOnSurfaceProcess(KM.Process):
```

that would be more consistent with the LOAD conditions